CHANGELOG-5.8.md

```diff
@@ -2,6 +2,18 @@
 
 ## [Unreleased](https://github.com/laravel/framework/compare/v5.8.27...5.8)
```

Comment:
Is it not 5.8.28?

Reply:
not yet ) It will be 5.8.28 when we will have next release)